app/src/main/java/org/simple/clinic/home/overdue/search/OverdueSearchScreen.kt

```diff
@@ -310,6 +321,14 @@ class OverdueSearchScreen : BaseScreen<
     selectedOverdueCountView.visibility = GONE
   }
 
+  override fun showSelectAllLayout() {
+    selectAllLayout.visibility = VISIBLE
+  }
```

Comment:
We need to show the select all button only if the select & download feature is enabled and if country is India. It doesn't look like we are covering that case.